modules/core/src/test/java/org/apache/ignite/internal/testframework/VariableArgumentsProvider.java

```diff
@@ -45,7 +45,7 @@ public void accept(VariableSource variableSource) {
     public Stream<? extends Arguments> provideArguments(ExtensionContext context) throws Exception {
         Class<?> testClass = context.getRequiredTestClass();
 
-        Field source = testClass.getDeclaredField(variableName);
+        Field source = testClass.getField(variableName);
```

Comment:
Very interesting, but what was wrong with non-public fields?

Reply:
`getDeclaredField` doesn't look into fields that are declared in superclasses. It could be fixed by manually traversing the hierarchy, but I don't see anything wrong with simply declaring these variables as `public`

Reply:
Ok, I realized that after commenting. It's fine